source/connect-to-mongo/tls.txt

```diff
@@ -17,11 +17,27 @@ TLS/SSL
    :depth: 2
    :class: singlecol
 
+Overview
+--------
+
+In this guide, you can learn how to use the **TLS protocol** to secure your
+connection to a MongoDB deployment. TLS is a cryptographic protocol that
+secures communication between your application and MongoDB. To configure
+your connection to use TLS, enable the TLS option and provide your
+certificates for validation when creating a client.
+
 By default, the driver supports TLS/SSL connections to MongoDB
 servers using the underlying support for TLS/SSL provided by the JDK.
-This can be changed either by utilizing the extensibility of the `Java SE
-API <https://docs.oracle.com/javase/8/docs/api/>`__, or by using the
-`Netty API <https://netty.io/4.1/api/>`__.
+This can be changed either by using the `Netty API
+<https://netty.io/4.1/api/>`__ or the extensibility of the `Java SE
+API <https://docs.oracle.com/javase/8/docs/api/>`__.
+
+.. tip:: Prefer Netty for Asynchronous Apps
+
+   We recommend using Netty for asychronous applications, as it supports
```

Comment:
[Style guide - as](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-as)
```suggestion
   We recommend using Netty for asychronous applications because it supports
```